search/docker-compose.yml

```diff
@@ -29,19 +77,22 @@ services:
       interval: 30s
       timeout: 30s
       retries: 3
+
   reactivesearch:
     image: my-reactivesearch
-    build: .
-    volumes:
-      - ./:/code # for development
-      - /code/node_modules # an exception from the above volume
+    container_name: reactivesearch
+    build:
+      context: .
+      dockerfile:  Dockerfile
     ports:
-      - "1337:3000"
+      - "8080:80"
     environment:
       - NODE_ENV=production
     networks:
       - elastic
     stdin_open: true
+    depends_on:
+      - elasticsearch
 
   # kibana as another elastic search UI
```

Comment:
I don't think we have any need for this in prod

Reply:
Do we need it if it's protected by nginx auth? or we just don't need it at all.

Reply:
I don't think we need kibana at all, though it might be useful to play around with the data occasionally in dev... In that case we can even launch it as a separate process without it being in compose, tbh. I think it's time to stop worrying about a kibana that we just don't use.